### PR TITLE
Move actors description to docstring

### PR DESCRIPTION
**Pull request by artmello**

### CODE REVIEW

**Comment by vinzenz** [comment]
leapp-ci build

**Comment by vinzenz** [comment]
leapp-ci build

**Comment by pirat89** [comment]
leapp-ci build

**Comment by pirat89** [comment]
leapp-ci build

File: repos/system_upgrade/el7toel8/actors/checkbtrfs/actor.py [hunk 1]
```diff
@@ -4,8 +4,14 @@
 
 
 class CheckBtrfs(Actor):
+    """
+    Check if Btrfs filesystem is in use. If yes, inhibit the upgrade process.
+
+    Btrfs filesystem was introduced as Technology Preview with initial releases of RHEL 6 and 7. It
+    was deprecated on versions 6.6 and 7.4 and will not be ṕresent in next major version.
```
Comment (pirat89) [inline repos/system_upgrade/el7toel8/actors/checkbtrfs/actor.py]:
```suggestion
    was deprecated on versions 6.6 and 7.4 and will not be present in next major version.
```

unicode :dagger:  :D

**Pull request merged**